go-bridge-api/entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env sh
+set -e
+
+ACCESS_TOKEN="${ACCESS_TOKEN:-access_token}"
+ACCOUNT_KEY="${ACCOUNT_KEY:-account_key}"
+CONVERSATION_KEY="${CONVERSATION_KEY:-conversation_key}"
+TRANSPORT_NAME="${TRANSPORT_NAME:-transport_name}"
+WEB_PATH="${WEB_PATH:-/go-bridge-api}"
+BASE_URL="${BASE_URL:-https://go.vumi.org/api/v1/go/http_api_nostream/}"
+REDIS_HOST="${REDIS_HOST:-127.0.0.1}"
+REDIS_DB="${REDIS_DB:-1}"
+REDIS_PORT="${REDIS_PORT:-6379}"
+AMQP_HOST="${AMQP_HOST:-127.0.0.1}"
+AMQP_PORT="${AMQP_PORT:-5672}"
+AMQP_USERNAME="${AMQP_USERNAME:-guest}"
+AMQP_PASSWORD="${AMQP_PASSWORD:-guest}"
+AMQP_VHOST="${AMQP_VHOST:-/guest}"
+AMQP_PREFETCH_COUNT="${AMQP_PREFETCH_COUNT:-20}"
+
+
+cat > /app/config.yml <<EOF
+# Generated config file
+access_token: ${ACCESS_TOKEN}
+account_key: ${ACCOUNT_KEY}
+conversation_key: ${CONVERSATION_KEY}
+transport_name: ${TRANSPORT_NAME}
+web_port: 8000
+web_path: ${WEB_PATH}
+base_url: ${BASE_URL}
+amqp_prefetch_count: ${AMQP_PREFETCH_COUNT}
+
+redis_manager:
+  host: ${REDIS_HOST}
+  port: ${REDIS_PORT}
+  db: ${REDIS_DB}
+
+EOF
+CONFIG_FILE="/app/config.yml"
+
```

Comment:
Maybe leave out the `AMQP_HOST`...`AMQP_PASSWORD` stuff as well as this twisted plugin stuff and just call the Vumi entrypoint script?
```shell
...
export WORKER_CLASS="vumi.transports.vumi_bridge.GoConversationTransport"
export CONFIG_FILE="/app/config.yml"

exec vumi-entrypoint.sh
```